meeting-notes/2019/Q1/2019-01-03--ipfs-community-wg-weekly.md

```diff
@@ -0,0 +1,76 @@
+# IPFS Community Working Group Weekly Sync 2019-01-03
+
+-   **Moderator:** @terichadbourne
+-   **Notetaker:** @pkafei
+-   **Attendees:**
+    -   @pkafei
+    -   @jimpick
+    -   @terichadbourne
+    -   @mikeal
+    -   @NukeManDan
+
+**Moderator checklist**
+-   Ensure that there is a notetaker
+-   Start recording
```

Comment:
Where can I find the recording?

Reply:
@daviddias On my laptop in Boston! I don't have access to the YouTube account to upload it and haven't yet heard back from it@protocol.ai on how to get access.

Reply:
@terichadbourne just checked if I still had admin and managed to invite you. You should have an email on your inbox :)

Reply:
<img width="213" alt="image" src="https://user-images.githubusercontent.com/1211152/50780544-21559b00-129b-11e9-9dee-c1439789d89b.png">

Look at @mikeal so happy! :D

Reply:
*protip - recording will pick up on your view. If you set "gallery view" by default, it will default to show everyone's faces, making the call feel more full

Reply:
@daviddias Thanks for the help getting access to the channel. I'm going to review Molly's video tips before the next call and try to figure out how to set up the livestream. 

I've added the link to the recording to the meeting notes. 

What's the typical approach for merging PRs of meeting notes? Shall I go ahead and hit merge myself after leaving an issue up for a few days and addressing any concerns expressed?

Reply:
> Shall I go ahead and hit merge myself after leaving an issue up for a few days and addressing any concerns expressed?

That's pretty much it. The great thing about PRs is that folks can always go back to it at any time if they want to ask any extra question :)